docs/indexer/build/subgraph-style.md

```diff
@@ -0,0 +1,239 @@
+# Comparison with Subgraph Query Style.
+We support using `Subgraph` query style `GraphQL` queries, which are a bit different from `subquery` style. 
+Below is an introduction to how to install it and the differences between it and subql-query.
+
+## Install subql/query-subgraph
+Then to run a SubQuery node, run the following command:
+
+```shell
+npm install -g @subql/query-subgraph
+```
+
+The `-g` flag means to install it globally which means on OSX, the location will be `/usr/local/lib/node_modules`.
+
+Once installed, you can check the version by running:
+
+```shell
+> subql-query-subgraph --version
+0.1.0
+```
```

Comment:
Can you please add examples for docker and docker compose, also updating the query runner in project manifests